packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -502,9 +531,35 @@ export class TempCollabSpaceRuntime
 	public async getCellDebugInfo(
 		row: number,
 		col: number,
-	): Promise<{ channel?: ICollabChannelCore }> {
+	): Promise<{ channel?: ICollabChannelCore; channelId?: string; rowId: string; colId: string }> {
 		const result = this.getCellInfo(row, col);
-		return { channel: await result.channel };
+		assert(result.channelId !== undefined, "channelId is missing");
+		const { rowId, colId } = this.parseChannelId(result.channelId);
+		const channel = await result.channel;
+		return { channel, channelId: result.channelId, rowId, colId };
+	}
+
+	public getReverseMapCellDebugInfo(rowId: string, colId: string): { row: number; col: number } {
+		const row = this.reverseMap.getRowId(rowId);
```

Comment:
API name does not make sense. It returns index, not rowId, so it should not be called getRowId. Same for getColId